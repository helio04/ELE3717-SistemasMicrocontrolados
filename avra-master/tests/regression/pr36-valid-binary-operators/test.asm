.device atmega2560

ldi r16, 24 * 3
ldi r16, 24 / 3
ldi r16, 24 % 5
ldi r16, 24 + 3
ldi r16, 24 - 3
ldi r16, 24 < 3
ldi r16, 24 < 24
ldi r16, 3 < 24
ldi r16, 24 << 3
ldi r16, 24 <= 3
ldi r16, 24 <= 24
ldi r16, 3 <= 24
ldi r16, 24 > 3
ldi r16, 24 > 24
ldi r16, 3 > 24
ldi r16, 24 >> 2
ldi r16, 24 >= 3
ldi r16, 24 >= 24
ldi r16, 3 >= 24
ldi r16, 24 == 3
ldi r16, 24 == 24
ldi r16, 3 == 24
ldi r16, 24 != 3
ldi r16, 24 != 24
ldi r16, 3 != 24
ldi r16, 56 & 15
ldi r16, 1 && 1
ldi r16, 1 && 0
ldi r16, 0 && 1
ldi r16, 0 && 0
ldi r16, 24 ^ 9
ldi r16, 24 | 11
ldi r16, 1 || 1
ldi r16, 1 || 0
ldi r16, 0 || 1
ldi r16, 0 || 0
