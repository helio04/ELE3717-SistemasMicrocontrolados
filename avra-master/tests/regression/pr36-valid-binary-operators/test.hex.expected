:020000020000FC
:1000000008E408E004E00BE105E100E000E001E0C5
:1000100000EC00E001E001E001E000E000E006E0CB
:1000200001E001E000E000E001E000E001E000E0CC
:1000300001E008E001E000E000E000E001E10BE1A8
:0800400001E001E001E000E035
:00000001FF
